runtime/onert/core/include/util/OnertException.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_UTIL_ONERTEXCEPTION_H__
+#define __ONERT_UTIL_ONERTEXCEPTION_H__
+
+#include <iostream>
+
+namespace onert
+{
+namespace util
+{
+
+class OnertException : public std::exception
+{
+  std::string message;
+
+public:
+  OnertException(std::string _m) : message(_m) {}
+
+  virtual const char *what() const throw() { return message.c_str(); }
+};
+
+class ExecutionFailedException : public OnertException
+{
+public:
+  ExecutionFailedException(std::string _m) : OnertException("ExecutionFailedException : " + _m) {}
+};
+
+class FileNotFoundException : public OnertException
+{
+public:
+  FileNotFoundException(std::string _m) : OnertException("FileNotFoundException : " + _m) {}
+};
+
+class NotSupportedTypeException : public OnertException
```

Comment:
case 1) Some datatype is not supported (e.g., int128 :-P , meaning out of spec)
case 2) but some is not yet supported (e.g., int32 support for some ops).
I wonder if this is this for case 1).
Should we throw `NotYetImplementedException` in case of case 2)?
It's somewhat confusing which one to throw...

Reply:
add NotSupportedTypeException and NotYetSupportedTypeException